profile/README.md

```diff
@@ -1,14 +1,22 @@
 # Welcome to JuliaGPs
 
 JuliaGPs is an organisation interested in making Gaussian process models work well in the [Julia programming language](https://julialang.org/).
-A lot of us use [Gaussian processes](https://en.wikipedia.org/wiki/Gaussian_process) in our research, or do methodological research on Gaussian processes.
+The packages in this ecosystem are targeted at people who want to use [Gaussian processes](https://en.wikipedia.org/wiki/Gaussian_process) as Bayesian statistical models,
+or people who want to do methodological research on Gaussian processes.
 
 If you're new to the organisation, you should develop an understanding of the core packages:
 1. [KernelFunctions.jl](https://github.com/JuliaGaussianProcesses/KernelFunctions.jl)
-2. [AbstractGPs.jl](https://github.com/JuliaGaussianProcesses/AbstractGPs.jl)
-3. [ApproximateGPs.jl](https://github.com/JuliaGaussianProcesses/ApproximateGPs.jl)
+2. [GPLikelihoods.jl](https://github.com/JuliaGaussianProcesses/GPLikelihoods.jl)
+3. [AbstractGPs.jl](https://github.com/JuliaGaussianProcesses/AbstractGPs.jl)
+4. [ApproximateGPs.jl](https://github.com/JuliaGaussianProcesses/ApproximateGPs.jl)
+
+`KernelFunctions` and `GPLikelihoods` are low-level packages implementing APIs for kernel functions and observation likelihoods, respectively, and include implementations of the most common
+classes of kernels and likelihoods used in pratice.
+`AbstractGPs` and `ApproximateGPs` are higher-level packages that implement inference of both full and sparse GPs with Gaussian and non-Gaussian likelihoods, respectively.
```

Comment:
I would keep these two points separate, especially since the respectively refers to 3 points.
```
ERROR: DimensionMismatch: argument dimensions must match
```
:laughing:

Reply:
Bump

Reply:
I think it needs a less ambiguous formulation (it's not clear if the respectively applies to sparse GPs for Gauss. and non-Gaussian, or if both have full and sparse.

Something like:

```suggestion
`AbstractGPs` and `ApproximateGPs` are higher-level packages that implement inference of full and sparse GPs. `AbstractGPs` is restricted to Gaussian likelihoods, while `ApproximateGPs` also allows for non-Gaussian ones.
```

?